### PR TITLE
Fix single package features in a workspace

### LINKED ISSUE #1516: Make features work the way as in `cargo doc`
As for now, we strictly check whether the sub packages have the features from args inside the manifests. We want to make it work as in `cargo doc`. We want to ignore them (if they are missing in manifest), rather than throwing an error.

### PR DESCRIPTION
**Pull request by maciektr**
fix #1516 #1599

### CODE REVIEW

**Comment by maciektr** [comment]
@wawel37 should this fix https://github.com/software-mansion/scarb/issues/1516?

File: scarb/src/ops/resolve.rs [hunk 1]
```diff
@@ -207,6 +212,25 @@ pub fn generate_compilation_units(
...
+pub fn validate_features(members: &[Package], enabled_features: &FeaturesOpts) -> Result<()> {
+    // Check if any member has features defined.
+    if let FeaturesSelector::Features(features) = &enabled_features.features {
+        for feature in features {
+            if !members
+                .iter()
+                .any(|member| member.manifest.features.contains_key(feature))
+            {
+                bail!(
+                    "none of selected packages contains `{}` feature\n\
```
Comment (wawel37) [inline scarb/src/ops/resolve.rs:224]:
```suggestion
                    "none of the selected packages contains `{}` feature\n\
```

**Review by wawel37 — APPROVED** [review]
LGTM

**Comment by wawel37** [comment]
> @wawel37 should this fix #1516?

Yup! If we check for any member with given feature, it will work for sure!

**Pull request merged**